include/WCSimDetectorConstruction.hh

```diff
@@ -80,7 +82,8 @@ public:
   G4int    GetMyConfiguration()   {return myConfiguration;}
   G4double GetGeo_Dm(G4int);
   G4int    GetTotalNumPmts() {return totalNumPMTs;}
-  
+  G4int    GetTotalNumPmtsOD() {return totalNumPMTsOD;}
```

Comment:
… and similarly for `WCPMTODName` ➔ `WCODPMTName` and several others throughout this file. This is not about the name/radius/… of the “PMT outer detector” but about the “outer detector PMTs”, so I think consistently calling them `ODPMT` in all variable/function names will make the code slightly easier to read and modify later on.